frontend/src/components/Posts/FilterContainer.tsx

```diff
@@ -114,13 +120,25 @@ export const FilterContainer = ({
       } else {
         // Else, remove "all" option from selected array and add selectedOption
         const filtered = (selected as Option[]).filter(
-          (option) => option.value !== "all"
+          (option) => option.value !== "all",
         );
         setSelected([...filtered, selectedOption]);
       }
     }
   };
 
+  // don't show "requests" filter on individual user's "my posts" tab
+  const hideFiltersFromIndividual = (postType: string) => {
```

Comment:
I think this should be renamed to match the boolean return type.

if `hideFiltersFromIndividual()` returns true, then this component will render.
if `hideFiltersFromIndividual()` returns false, then this component will not render the filter option.

To me, that kind of contradicts the promise between variable name and output.

What if we renamed it to something like `showFilterForIndividual()`?